craigslist/base.py

```diff
@@ -95,14 +95,14 @@ def get_filters(self, filters):
                            list_filters[key])
                 if filter_['value'] is None:
                     parsed_filters[filter_['url_key']] = value
-                elif isinstance(filter_['value'], list):
+                elif isinstance(filter_['value'], dict):
                     valid_options = filter_['value']
-                    if not hasattr(value, '__iter__'):
+                    if not hasattr(value, '__iter__') or isinstance(value, str):
                         value = [value]  # Force to list
                     options = []
                     for opt in value:
                         try:
-                            options.append(valid_options.index(opt) + 1)
+                            options.append(valid_options.get(opt))
                         except ValueError:
```

Comment:
Let's use square brackets instead of `get` here to force this to fail. This would then be caught by the except `KeyError`:

```suggestion
                            options.append(valid_options[opt])
                        except KeyError:
```